framework/PostProcessors/Metric.py

```diff
@@ -238,16 +237,36 @@ def _writeXML(self,output,outputDictionary):
             self.raiseAnError(IOError, "Unrecognized type of input value '", type(value), "'")
     outputInstance.writeFile()
 
-  #def _writeText(self,output,outputDictionary, separator=' '):
-  #  """
-  #    Defines the method for writing the post-processor to a .csv file
-  #    @ In, output, File object, file to write to
-  #    @ In, outputDictionary, dict, dictionary stores importance ranking outputs
-  #    @ In, separator, string, optional, separator string
-  #    @ Out, None
-  #  """
-    ##TODO: Add this method if needed
-    #self.raiseAnError(IOError, "Write output files into a text file is not implemented yet!")
+  def _writeText(self, output, outputDictionary, separator=' '):
+    """
+      Defines the method for writing the post-processor to a .csv file
+      @ In, output, File object, file to write to
+      @ In, outputDictionary, dict, dictionary stores importance ranking outputs
```

Comment:
importance ranking outputs?

Reply:
Forgot to update the comments, copy and paste error.